src/software/Localization/main_rigLocalizer.cpp

```diff
@@ -122,27 +122,49 @@ int main(int argc, char** argv)
-          ("results,r", po::value<size_t>(&numResults)->default_value(numResults), "Number of images to retrieve in database")
+        ("help,h", "Print this message")
+        ("descriptors,", po::value<std::string>(&str_descriptorType)->default_value(str_descriptorType), 
+          "Type of descriptors to use {SIFT,CCTAG,SIFT_CCTAG}")
+        ("preset,", po::value<std::string>(&preset)->default_value(preset), 
+          "Preset for the feature extractor when localizing a new image {LOW,NORMAL,HIGH,ULTRA}")
+        ("sfmdata,d", po::value<std::string>(&sfmFilePath)->required(),
+            "The sfm_data.json kind of file generated by OpenMVG [it could be "
+            "also a bundle.out to use an older version of OpenMVG]")
+        ("siftPath,s", po::value<std::string>(&descriptorsFolder)->required(),
+            "Folder containing the .desc [for the older version of openMVG it is "
+            "the list.txt].")
+        ("mediapath,m", po::value<std::string>(&mediaPath)->required(),
+            "The folder path containing all the synchronised image subfolders "
+            "assocated to each camera")
+        ("filelist", po::value<std::string>(&filelist),
+            "An optional txt file containing the images to use for calibration. "
+            "This file must have the same name in each camera folder and contains "
+            "the list of images to load.")
+        ("refineIntrinsics,", po::bool_switch(&refineIntrinsics),
+            "Enable/Disable camera intrinsics refinement for each localized image")
+        ("nCameras", po::value<size_t>(&numCameras)->default_value(numCameras), 
+          "Number of cameras composing the rig")
+        ("calibration,c", po::value<std::string>(&rigCalibPath)->required(), 
+          "The file containing the calibration data for the rig (subposes)")
+// parameters for voctree localizer
+        ("voctree,t", po::value<std::string>(&vocTreeFilepath),
+          "[voctree] Filename for the vocabulary tree")
+        ("weights,w", po::value<std::string>(&weightsFilepath),
+          "[voctree] Filename for the vocabulary tree weights")
+        ("algorithm,", po::value<std::string>(&algostring)->default_value(algostring),
+          "[voctree] Algorithm type: {FirstBest,BestResult,AllResults,Cluster}" )
+        ("results,r", po::value<size_t>(&numResults)->default_value(numResults),
+          "[voctree] Number of images to retrieve in database")
 #if HAVE_CCTAG
   // parameters for cctag localizer
-          ("nNearestKeyFrames", po::value<size_t>(&nNearestKeyFrames)->default_value(nNearestKeyFrames), "Number of images to retrieve in database")
+        ("nNearestKeyFrames", po::value<size_t>(&nNearestKeyFrames)->default_value(nNearestKeyFrames),
+          "[cctag] Number of images to retrieve in database")
 #endif
 #if HAVE_ALEMBIC
```

Comment:
nope, old description from a bad copy paste. It exports just an abc if supported.